source/css/daily.css

```diff
@@ -549,7 +549,7 @@ input {
 }
 
 #date-input {
-  width: 10vw;
+  width: fit-content;
```

Comment:
I think the width property here actually doesn't matter? I used grid for the layout and flex 2 for much space it takes so I am just wondering did this width property actually changed anything?

Reply:
I noticed when I was using my laptop (which has a smaller screen) the date input got cut off, so setting it to `fit-content` fixed it. I think on bigger screens there's no issue though.

Reply:
oh great that's awesome